<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tenduke::qt::demo::UserInfoWindow</class>
 <widget class="QMainWindow" name="tenduke::qt::demo::UserInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLoadUserinfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load userinfo</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblUserinfo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>601</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
